qubes_config/global_config.glade

```diff
@@ -7143,8 +7048,8 @@ to Global Clipboard</property>
                                   <object class="GtkLabel" id="thisdevice_certified_yes_info">
                                     <property name="visible">True</property>
                                     <property name="can-focus">True</property>
-                                    <property name="label" translatable="yes">&lt;b&gt;This device is certified for Qubes OS.&lt;/b&gt;
```

Comment:
I don't want users to fall into "once certified, always certified" fallacy from reading this, so I think it's important to qualify the statement. It's important for people to remember that each act of hardware certification is for a *specific* Qubes OS release.

Reply:
Do you propose something like "certified for QubesOS 4.2?"

Reply:
Either that or what I changed it to:

https://github.com/QubesOS/qubes-desktop-linux-manager/pull/150/files/639662da332b8ebea5a75f33154f7ec84c14adb7#diff-2316c0cc993b8ed282946d88086c3b571db8973b6805c5848a89524b83c7a106R7051

"This device is officially certified for your current Qubes OS release."

Which way is easier depends on how the check is coded. This way might not require any new/additional coding.

Reply:
Forgot to mention: Devices are certified for a specific *major* Qubes release, so if we do decide to mention a release or version number, it should probably be "4.X" (or similar) rather than "4.2" (which is a minor release).

(Of course, it's true that any machine certified for 4.X is thereby certified for 4.2, but this might be misleading in its overspecificity.)